### ISSUE TITLE
No Sync from Brave Browser on 3 PCs all with v4.19.0

### ISSUE THREAD

**Issue by dsiminiuk** [floccusaddon/floccus#1364]
### Which version of floccus are you using?

4.19.0

### Sync method

Nextcloud Bookmarks

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

Brave

### Which version of Nextcloud Bookmarks are you using? (if relevant)

12.1.0

### Which version of Nextcloud? (if relevant)

25.0.5

### Describe the Bug

No sync up or down even when I add or remove bookmarks from the NC Booksmarks app. I've uninstalled, reinstalled, no differnce.

### Expected Behavior

It should sync like it used to.

### To Reproduce

Click the Floccus add-in button and sync, push up or push down, no sync. One laptop with Brave now shows "Disabled". ???

### Debug log provided

- [X] I have provided a debug log file

**Comment by dsiminiuk** [comment]
I reverted to v4.18.1 and it works fine again.

**Comment by BOFH90** [comment]
Same issue on macOS and Windows with Nextcloud, Brave and v4.19.0

**Comment by dsiminiuk** [comment]
@baritell Agreed! I played with 4.19.0 some more on Windows and it does work for bookmarks in folders below the Bookmark Bar but not for changes on the bookmark bar. Good find!

**Comment by marcelklehr** [comment]
Thanks for reporting this. I have a fix in mind but don't know when I'll get to applying and releasing it. Fingers crossed

**Comment by dsiminiuk** [comment]
@marcelklehr 

Just confirmed on Windows 10, Brave Browser and floccus 4.19.0 - when adding a new bookmark to the bookmark bar, it does not upload it to the server. Move the same bookmark to a folder and voila!

**Comment by cimenta** [comment]
> same issue here. reverting back to v4.18.1 solved and sync works.

could you please provide steps how to revert?

 I downloaded crx file of v4.18.1 file from https://github.com/floccusaddon/floccus/releases but got this error https://i.imgur.com/z3QivOo.png

**Comment by eihnat** [comment]
> 

Managed to do this only in Firefox.
https://addons.mozilla.org/en-US/firefox/addon/floccus/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search
on extension page there's on left lower side "see all versions" and you can pick which one you want. you need to uninstall newer one first i think.
Didn't find how to do it on Brave (Chrome probably same).

**Comment by dsiminiuk** [comment]
I was a Vivaldi user until recently. They had a bug that wiped all my bookmarks so I jumped to Brave. That was fine until this bug in 4.19.0 showed up. And even then it is OK on Linux. You can go to your Extensions page and enable Developer Mode, export your settings from floccus, remove it, download the crx file for the release you want, and drop it on the Developer Mode Extensions page.
But with the Windows version of Brave, it won't let you install a crx extension for any version different than the one currently published on the Chrome Webstore. Here is a thread about that https://github.com/brave/brave-browser/issues/20296.

Last night I tried Midori - didn't like it's lack of features, search engine limitations specifically. Then I tried SRWare Iron, and it allowed me to install the crx file the usual way, but when I did a forced download of my bookmarks they didn't appear on my Bookmarks bar. Perhaps I could have figured out what was happening there but it was late and I'd had enough messing around.

One workaround might be for @marcelklehr to pull the v4.19.0 off the Chrome Store and re-publish v4.18.1 so I could get that version back on my Windows/Brave machines. *cough*cough*

**Comment by marcelklehr** [comment]
>  it does work for bookmarks in folders below the Bookmark Bar but not for changes on the bookmark bar. 

Which folder do you have selected as your local sync target in this scenario?

**Comment by dsiminiuk** [comment]
@marcelklehr Here is a snip...
![2023-03-29_10-18](https://user-images.githubusercontent.com/5713547/228587006-431e9ef5-3316-4a18-8bcb-c2b26a696046.png)

**Comment by marcelklehr** [comment]
Thanks!

**Comment by baritell** [comment]
For me I have it set to `/Bookmarks Toolbar/`.  Just fyi, in case it matters.  Thanks for your help!
![image](https://user-images.githubusercontent.com/22551714/228589160-b5f80ea0-7795-4272-a8b3-e9277e221571.png)

**Comment by marcelklehr** [comment]
@baritell Thanks for confirming, this affects everyone syncing the main / built-in folders (but not the absolute untitled root folder).

**Comment by dsiminiuk** [comment]
@baritell The name of it is browser dependent.

**Comment by marcelklehr** [comment]
It seems I'm unable to publish a new release without having to push out a patch a few days later ... :see_no_evil: Preparing a release with the fix as we speak

**Comment by marcelklehr** [comment]
Should be fixed with v4.19.1 -- coming to a machine near you soon-ish.

**Comment by dsiminiuk** [comment]
@marcelklehr Now that v4.19.1 is available on the Chrome Webstore I was able to install it on the WIndows/Brave PCs and all is good. I did remove the 4.19.0 and add back 4.19.1 importing a previously exported .json file and then did a one time pull initially, now everything syncs perfectly.

Thank you for your quick action. This extension (and Bitwarden for passwords) keep me sane. Another contribution is on the way.
